docs/src/rules/no-else-return.md

```diff
@@ -59,7 +59,7 @@ function foo3() {
     if (x) {
         return y;
     } else {
-        var t = "foo";
+        const t = "foo";
```

Comment:
In this example, it looks like `var` makes more sense than `const`?

Reply:
changed!